about.html

```diff
@@ -0,0 +1,227 @@
+        </p>
+        <div class="past-itmes-cards">
+          <div class="card-1">
+            <img src="assets/images/watch.jpg" alt="past sold item" />
+            <div class="layer">
+              <h3>
+                2020 <br />
+                Men's Watches
+              </h3>
+            </div>
+          </div>
+          <div class="card-2">
+            <img src="assets/images/shoes.jpg" alt="past sold item" />
+            <div class="layer">
+              <h3>2021 <br />Men's Shoes</h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-partners about-brandsLogo">
+      <div class="prodContainer">
+        <div class="container">
+          <h3 class="title">Our Partners</h3>
+          <div class="row">
+            <div class="col-5">
+              <img src="assets/images/logo-godrej.png" alt="godrej logo" />
+            </div>
+            <div class="col-5">
+              <img src="assets/images/logo-oppo.png" alt="oppo logo" />
+            </div>
+            <div class="col-5">
+              <img
+                src="assets/images/logo-coca-cola.png"
+                alt="coca-cola logo"
+              />
+            </div>
+            <div class="row-2">
+              <div class="col-5">
+                <img src="assets/images/logo-paypal.png" alt="paypal logo" />
+              </div>
+              <div class="col-5">
+                <img src="assets/images/logo-philips.png" alt="philips logo" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer-about">
+      <div class="container">
+        <div class="foot-left">
+          <a href="index.html"
+            ><img src="assets/images/yaycommerceNobg.png" alt="logo"
+          /></a>
+        </div>
+        <div class="foot-right">
+          <h4>
+            Copyright &copy; 2022 GeezTech Systems, Inc. All Rights Reserved
+          </h4>
+          <ul>
+            <li><a href="#" class="privacy">Privacy Policy</a></li>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Accessibility</a></li>
+            <li><a href="#">Sitemap</a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
+    <footer class="foot-about-mobile">
+      <div class="container">
+        <div class="foot-left">
+          <a href="index.html"
+            ><img src="assets/images/yaycommerceNobg.png" alt="logo"
+          /></a>
+        </div>
+        <div class="foot-right">
+          <h4>
+            Copyright &copy; 2022 GeezTech Systems, Inc. All Rights Reserved
+          </h4>
+          <ul>
+            <li><a href="#" class="privacy">Privacy Policy</a></li>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Accessibility</a></li>
+            <li><a href="#">Sitemap</a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
```

Comment:
Our partners and Footer in the mobile version aren't responsive. As you can see the logos in Our Partners section is overflowing and the footer link isn't clearly visible. Kindly make it responsive as well.

<img width="315" alt="Screen Shot 2022-04-15 at 5 23 57 AM" src="https://user-images.githubusercontent.com/83048208/163496767-d7bf6376-2737-4b53-be83-ebaf10f85cd3.png">